Source/WebKit/UIProcess/BackingStore.h

```diff
@@ -61,11 +63,11 @@ class BackingStore {
 
     WebCore::IntSize m_size;
     float m_deviceScaleFactor { 1 };
-    WebCore::PlatformImagePtr m_surface;
-    WebCore::PlatformImagePtr m_scrollSurface;
+    RefPtr<cairo_surface_t> m_surface;
+    RefPtr<cairo_surface_t> m_scrollSurface;
```

Comment:
Please do a `PLATFORM(GTK)` check here and don't remove the `WebCore::PlatformImagePtr`. This will break some of the porting I'm doing for other platforms to use Skia.

If possible just modify the `PlatformImagePtr` definition.

Reply:
Much simpler!